lib/src/builder.dart

```diff
@@ -1247,7 +1248,8 @@ class _MockClassInfo {
   ///
   /// This is very similar to Angular's revive code, in
   /// angular_compiler/analyzer/di/injector.dart.
-  Expression _expressionFromDartObject(DartObject object) {
+  Expression _expressionFromDartObject(DartObject object,
+      [ParameterElement? parameter]) {
```

Comment:
Would you mind documenting on the line above, what `parameter` represents?

Reply:
Done